wiz-broker/values.yaml

```diff
@@ -57,18 +57,18 @@ wizConnector:
   annotations: {}
   secretName: ""
 
-  connectorId: ""
-  connectorToken: ""
-  targetDomain: ""
-  targetIp: ""
-  targetPort: ""
-  tunnelServerDomain: ""
-  tunnelServerPort: ""
+  connectorId: "<required-connector-id>"
```

Comment:
We don't want this as default value, We prefer that the helm template will fail and ask you to fill this values...
Please revert all your values changes